### PR TITLE
Some corrections to work with the wrapper class

### PR DESCRIPTION
**Pull request by dkazanc**
Should be used in conjunction with this [PR](https://github.com/DiamondLightSource/httomo/pull/74) for testing

### CODE REVIEW

**Comment by dkazanc** [comment]
tests now pass, merging.

File: httomolib/recon/algorithm.py [hunk 1]
```diff
@@ -57,42 +56,23 @@ def reconstruct_tomobar(
...
-        # performs 3D FBP with filtering on the CPU (host (filtering) -> device (backprojection) -> host (if needed))
-                
-        reconstruction = RectoolsDIR.FBP(np.swapaxes(data, 0, 1)) # the output stored as a numpy array
-        
-        # ------------------------------------------------------- #     
-    elif algorithm == "FBP3D_device":
-        # Perform filtering of the data on the GPU and then pass a pointer to CuPy array to do backprojection, i.e.
-        # (host -> device (filtering) -> device (backprojection) -> host (if needed))
         
+        # Perform filtering of the data on the GPU and then pass a pointer to CuPy array to do backprojection.
```
Comment (namannimmo10) [inline httomolib/recon/algorithm.py:75]: The indentation here is not correct.

File: tests/test_recon/test_recon.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import cupy as cp
+import numpy as np
+import pytest
+from httomolib.prep.normalize import normalize_cupy
+from httomolib.recon.rotation import find_center_360, find_center_vo_cupy
+from numpy.testing import assert_allclose
+
+
+@cp.testing.gpu
+def test_find_center_vo_cupy(data, flats, darks):
```
Comment (namannimmo10) [inline tests/test_recon/test_recon.py:10]: Why do we have this file? Duplicating tests from `tests/test_recon/test_rotation.py`.

File: httomolib/recon/algorithm.py [hunk 3]
```diff
@@ -57,42 +56,23 @@ def reconstruct_tomobar(
...
     gpu_id : int, optional
         A GPU device index to perform operation on.
 
     Returns
     -------
     cp.ndarray
-        The reconstructed volume as cp or np array.
+        The reconstructed volume as a CuPy array.
     """
     from tomobar.methodsDIR import RecToolsDIR
```
Comment (namannimmo10) [inline httomolib/recon/algorithm.py:67]: Unused import.

**Pull request merged**